tests/integration_tests/document_loaders/test_sitemap.py

```diff
@@ -9,11 +9,34 @@ def test_sitemap() -> None:
     assert "🦜🔗" in documents[0].page_content
 
 
+def test_sitemap_block() -> None:
+    """Test sitemap loader."""
+    loader = SitemapLoader("https://langchain.readthedocs.io/sitemap.xml", blocksize=1, blocknum=1)
+    documents = loader.load()
+    assert len(documents) == 1
+    assert "🦜🔗" in documents[0].page_content
+
+
+def test_sitemap_block_only_one() -> None:
+    """Test sitemap loader."""
+    loader = SitemapLoader("https://langchain.readthedocs.io/sitemap.xml", blocksize=1000000, blocknum=0)
+    documents = loader.load()
+    assert len(documents) > 1
+    assert "🦜🔗" in documents[0].page_content
+
+
+def test_sitemap_block_does_not_exists() -> None:
+    """Test sitemap loader."""
+    loader = SitemapLoader("https://langchain.readthedocs.io/sitemap.xml", blocksize=1000000, blocknum=15)
+    with pytest.raises(ValueError):
+        documents = loader.load()
```

Comment:
Might be a good idea to confirm the Error Message matches the expectation when the Value raise is intended. :)

You can find reference and examples in this link: https://docs.pytest.org/en/4.6.x/reference.html#pytest-raises

Reply:
will add the check for the error message